index.html

```diff
@@ -0,0 +1,320 @@
+                      hat. Dies passiert in der Regel fünf bis zehn Wochen,
+                      nachdem Sie Ihre Steuererklärung eingereicht haben. Bei
+                      größeren und schwierigeren Fällen oder in Zeiten, in denen
+                      besonders viele Leute ihre Erklärung abgeben, kann es
+                      jedoch gelegentlich länger dauern.
+                    </p>
+                    <p>
+                      Falls Sie sich über den Verbleib Ihres Steuerbescheids
+                      erkundigen wollen, ist Ihr zuständiges Finanzamt der
+                      richtige Ansprechpartner.
+                    </p>
+                  </div>
+                </div>
+              </div>
+              <div class="accordion-item">
+                <h2 class="accordion-header" id="flush-headingSix">
+                  <button
+                    class="accordion-button collapsed"
+                    type="button"
+                    data-bs-toggle="collapse"
+                    data-bs-target="#flush-collapseSix"
+                    aria-expanded="false"
+                    aria-controls="flush-collapseSix"
+                  >
+                    Ich habe noch eine Frage, an wen kann ich mich wenden?
+                  </button>
+                </h2>
+                <div
+                  id="flush-collapseSix"
+                  class="accordion-collapse collapse"
+                  aria-labelledby="flush-headingSix"
+                  data-bs-parent="#accordionFlushExample"
+                >
+                  <div class="accordion-body">
+                    Bei Fragen zum Steuerlotsen können Sie sich gerne an
+                    <a href="mailto:kontakt@steuerlotse.de"
+                      >kontakt@steuerlotse.de</a
+                    >
+                    wenden. Bei Fragen zu einfachELSTER finden Sie
+                    <a
+                      href="https://www.elster.de/eportal/infoseite/einfachelsterinfo"
+                      >auf dieser Seite</a
+                    >
+                    Kontaktmöglichkeiten und mehr Informationen.
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+        <footer class="pl-0 pr-0 ml-0 mr-0 mt-0">
+          <div class="footer-footer">
+              <div class="container-fluid">
+                  <div class="footer-inner">
+                      <div class="row">
+                          <a href="footer/privacy_statement.html">Datenschutzerklärung</a>
+                          <a href="footer/impressum.html">Impressum</a>
+                          <a href="footer/accessibility.html">Erklärung zur Barrierefreiheit</a>
+                      </div>
+                  </div>
+              </div>
+          </div>
+      </footer>
+      </div>
+      <script
+        src="https://cdn.jsdelivr.net/npm/@popperjs/core@2.9.2/dist/umd/popper.min.js"
```

Comment:
What are these javascript files used for? Is it related to plausible? Or is it just the accordion collapsing behaviour?